src/pages/content-history.js

```diff
@@ -105,7 +105,7 @@ module.exports = {
                                         m("a", {
                                                 title  : "Preview",
                                                 href   : m.route() + "/" + data.version,
-                                                config : m.route
+                                                target : "_blank"
```

Comment:
I like the consistency of previewing always opening an external link, but I dunno how useful that is on this one. Got an argument to sway me one way or the other?